src/run.js

```diff
@@ -32,9 +34,7 @@ module.exports = function run(regions, inputDir, outputDir) {
     inputFiles,
     processFile.bind(null, regions, inputDir, outputDir),
     function (code) {
-      if (code) {
-        process.exit(code);
-      }
+      process.exit(code || 0);
```

Comment:
I think you can get away with just passing `process.exit` as the callback, since it'll default to an exit code of 0 when `code === undefined`.